src/java/org/apache/cassandra/db/ReadExecutionController.java

```diff
@@ -243,4 +261,20 @@ private void addSample()
         if (cfs != null)
             cfs.metric.topLocalReadQueryTime.addSample(cql, timeMicros);
     }
+
+    public void addActivationIds(ColumnFamilyStore.ViewFragment view)
+    {
+        activationIds = new HashSet<>();
```

Comment:
possibly paranoid, but can we check this isn't null before creating a new hashset?